### ISSUE TITLE
Clean up Notes

### ISSUE THREAD

**Issue by sardred** [IT2810/it2810-webutvikling-h17-prosjekt-3-group-48#14]
Also, the Notes component has some really rough styling as of right now. I'm fine with the colors and all that, but the scrollbars per note are hard to avoid with material-ui. Also, they don't look editable at all, even though they're contentEditable. I suppose we don't need to remove that feature, but if we did, those scrollbars would go away... I think my dream layout here only shows note titles, but when you press them you get a modal or fill in a note-detail view with, well, the note details. Potentially you could have an "edit note" button like the delete button, that brings up a modal with fields for editing.
Also add in a "there's no body in the note because you didn't fill in a body in the note". Maybe make the tiny header larger and have it fill the space of the note then.
Also, I forgot to make a multiline text area for the note-body.

To summarize:
- [ ] Remove scrollbars
- [ ] Make editing more intuitive
- [ ] Remove unused space
- [ ] Multiline textarea
